admin_guide/out_of_resource_handling.adoc

```diff
@@ -60,6 +59,29 @@ When the node fails a pod, it terminates all containers in the pod, and
 the link:https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase[`PodPhase`] is transitioned to *Failed*.
 ====
 
+The node supports the `nodefs` and `imagefs` file system partitions when detecting disk pressure, as follows:
+
+* The `nodefs`, or `rootfs`, file system that the node uses for local disk volumes, daemon logs, emptyDir,
+and so on (for example, the file system that provides `/`). 
+* The `imagefs` file system that the container runtime uses for storing images and
+individual container-writable layers. Eviction thresholds are at 85% full for `imagefs`. The `imagefs` file system depends on the runtime and, in the case of Docker, which storage driver you are using:
++
+** For Docker, if you are using the `devicemapper` storage driver, the `imagefs` is thin pool. If you are using the `overlay2` storage driver, the `imagefs` is the file system that contains `/var/lib/docker/overlay2`.  
+The `rootfs` is the file system that contains `/var/lib/origin/openshift.local.volumes`.
++
+If you are using using the `devicemapper` storage driver, you can limit the read/write layer for the container using the `--storage-opt dm.basesize` flag in on the Docker daemon. 
++
+----
+$ sudo dockerd --storage-opt dm.basesize=50G
+----
+
+** For CRI-O, which uses overlay, the imagefs is *_/var/lib/containers/storage_* by default. 
+
+[NOTE]
+====
+If you are not using local storage isolation (ephemeral storage) and not using XFS quota (volumeConfig), there is no way to limit local disk usage by the pod. 
```

Comment:
s/are not using/do not use ... s/not using/do not use
s/there is no way to/you cannot